src/electron/views/results/components/command-bar.tsx

```diff
@@ -1,116 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-import { FlaggedComponent } from 'common/components/flagged-component';
-import { DropdownClickHandler } from 'common/dropdown-click-handler';
-import { NamedFC } from 'common/react/named-fc';
-import { CardsViewModel } from 'common/types/store-data/card-view-model';
-import { FeatureFlagStoreData } from 'common/types/store-data/feature-flag-store-data';
-import { ScanMetadata } from 'common/types/store-data/unified-data-interface';
-import {
-    ReportExportComponent,
-    ReportExportComponentDeps,
-} from 'DetailsView/components/report-export-component';
-import { UnifiedFeatureFlags } from 'electron/common/unified-feature-flags';
-import { ScanActionCreator } from 'electron/flux/action-creator/scan-action-creator';
-import { ScanStatus } from 'electron/flux/types/scan-status';
-import { ScanStoreData } from 'electron/flux/types/scan-store-data';
-import * as React from 'react';
-import { ReportGenerator } from 'reports/report-generator';
-
-import { InsightsCommandButton } from 'common/components/controls/insights-command-button';
-import * as styles from './command-bar.scss';
```

Comment:
Can `command-bar.scss` be removed as well?

Reply:
On second look, it's reused by `reflow-command-bar.tsx`, so probably better to rename it to match the remaining file than to delete it